settings.gradle.kts

```diff
@@ -177,7 +177,8 @@ dependencyResolutionManagement {
               "swirlds-merkle",
               "swirlds-fcqueue",
               "swirlds-jasperdb",
-              "swirlds-virtualmap"))
+              "swirlds-virtualmap",
+              "swirlds-test-framework"))
```

Comment:
Can we import `swirlds-cli` so we can write PCLI plugins in services?

Reply:
I discussed with @david-bakin-sl , this will be done in a separate PR